discussion_agents/cog/agent/react.py

```diff
@@ -67,11 +73,16 @@ def __init__(
         self.docstore = docstore
         self.enc = enc
 
+        if type_benchmark == 'HotpotQA' or type_benchmark == 'FEVER':
```

Comment:
shouldn't have this check

the user should find their on benchmark and just pass their examples from whichever benchmark they choose to the examples variable